src/lastRevision.sh

```diff
@@ -16,6 +16,6 @@
 
 # Find the current revision, store it in a file
 FILE=$1
-LASTREV=`svn info | grep '^Revision' | sed -e 's/Revision: *//'`
+LASTREV=`git rev-parse HEAD | cut -c1-8`
```

Comment:
Hi @rakeskadr, usually the first 8 characters of commit are enough to identify the commit.

But I think putting the whole SHA-1 is nice, because the more context the better. Also, it simplifies the code even more (to substr the sha1 is annoying on windows).

Thanks for pointing out!